### PR TITLE
Post release 1.23.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/zowe-explorer/README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ More information:
 
 > Zowe Explorer is compatible only with Theia 1.18.0 or higher.
 
+> Zowe Explorer could experience possible unexpected behaviors with the latest Theia releases.
```
Comment (zFernand0) [inline packages/zowe-explorer/README.md:38]: Should we remove the spacing between the notes ?
Not sure if that caused problems in some other builds before.

**Review by zFernand0 — COMMENTED** [review]
Changed LGTM!
Left one comment 😋

File: packages/zowe-explorer/README.md (hunk 1 above)
Comment (JillieBeanSim) [reply packages/zowe-explorer/README.md:38]: it will get fixed once the other is in master then merged into this PR :slightly_smiling_face:

**Review by zFernand0 — APPROVED** [review]

**Review by crawr — APPROVED** [review]
LGTM!

**Pull request merged**